hh-dust-fe-be/Frontend/Controllers/DeviceRegister.cs

```diff
@@ -0,0 +1,30 @@
+using Microsoft.AspNetCore.Mvc;
+using Frontend.Controllers.DTOs;
+using System;
+using System.Security.Cryptography;
+using System.Text;
+
+namespace Frontend.Controllers
+{
+    [ApiController]
+    [Route("api/[controller]")]
+    public class DeviceRegister : ControllerBase
+    {
+        public DeviceRegister()
+        {
+        }
+
+        [Route("{id}/derivedKey")]
+        public DeviceRegisterDto derivedKey(string id)
+        {
+            byte[] masterKey = Convert.FromBase64String("8isrFI1sGsIlvvFSSFRiMfCNzv21fjbE/+ah/lSh3lF8e2YG1Te7w1KpZhJFFXJrqYKi9yegxkqIChbqOS9Egw==");
```

Comment:
`masterKey` seems like sensitive information, so it should not be in code, it should be securely stored server side (in Azure the easiest way is environment variable, for local development use [Secret manger](https://docs.microsoft.com/en-us/aspnet/core/security/app-secrets?view=aspnetcore-6.0&tabs=windows#secret-manager)) and then read it here using Configuration API. Important - now that it was revealed to the whole world it is compromised and we need a new `masterKey`.

Reply:
As the description of the PR states, this is just a dummy key for testing purposes. Actually is the same key the used as an example in the [Azure documentation](https://docs.microsoft.com/en-us/azure/iot-dps/concepts-symmetric-key-attestation?tabs=azure-cli).

The correct key management is to happen in another PR as per #54, where I'm going to follow your recommendations. Thank you very much for your insights.